modules/auxiliary/server/mysql_rogue.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python3
+
+# mysql rogue server requirements
+import socket
+import asyncore
+import asynchat
+import struct
+import logging
+import logging.handlers
+import codecs
+from argparse import ArgumentParser
+from enum import Enum
+import metasploit.module as module
+
+metadata = {
+    'name': 'mysqlrogue',
+    'description': '''
+        The script starts a MySQL server that requests and retrieves files from clients that connect to it.
+    ''',
+    'authors': [
+        'Moritz Nentwig'
+    ],
+    'date': '2021-09-03',
```

Comment:
I'm a bit confused about the disclosure date and CVEs. The CVE is for PHPKB. There were similar issues in Adminer a few years ago. The MySQL client file disclosure issue is old (Gift's rogue server was published 9 years ago). Does this module use the same technique?